src/Umbraco.Web.UI.Client/src/common/interceptors/culturerequest.interceptor.js

```diff
@@ -0,0 +1,27 @@
+﻿(function() {
+   'use strict';
+
+    /**
+     * Used to set the current client culture on all requests API requests
+     * @param {any} $q
+     * @param {any} $routeParams
+     */
+    function cultureRequestInterceptor($q, $routeParams) {
+        return {
+            //dealing with requests:
+            'request': function(config) {
+                var apiPattern = /\/umbraco\/backoffice\//;
```

Comment:
This can't be hard coded, the "umbraco" path is configurable, it's part of the server vars collection, pretty sure it's in the `applicationPath` key

Reply:
I also wonder if this should just target the EntityController endpoints since that's the only place it's used? That can be done by returning the value of `Umbraco.Sys.ServerVariables.umbracoUrls.entityApiBaseUrl`

Reply:
It's just a HTTP header, best add it to all requests and be sure any custom controllers down the road have same abilities as the core ones. 

I'll get right on it. If everything else fails I'll just look for /backoffice/ 😄

Reply:
Fixed in 825d842